sdk/storage/storage-blob-changefeed/src/AvroReaderFactory.ts

```diff
@@ -1,8 +1,8 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import type { AvroReadable } from "../../storage-internal-avro/src";
-import { AvroReader } from "../../storage-internal-avro/src";
+import type { AvroReadable } from "@azure/storage-internal-avro";
```

Comment:
Here is the same as storage-common. It may not be safe to expose it as a separate project for customer using.

Reply:
@EmmaZhu we can certainly add documentation to that effect that there are no guarantees with this project and is internal use only.

Reply:
Hi @mpodwysocki ,

I'm just worried and the document may not be able to prevent customer from using it.  Is there a guide document about how should we handle project only for internal using, like:
- We may need to release the internal project before the SDK packages for public using. What'd the release process look like? 
- If I need to change interface for internal using project, do I still need to go through review process? What process should I obey for a breaking change?

Reply:
We can have documentation to say that it is internal or you use the API docs for `@internal` which will also state that it is not intended for public usage.  If any customer is using these APIs, they are are not guaranteed to work and all bugs opened against them will be closed automatically.  And no if it's not part of the public API surface, it is not considered a breaking change especially when marked as internal.